osu.Game/Users/UserStatistics.cs

```diff
@@ -71,21 +71,21 @@ private int rank
         public struct Grades
         {
             [JsonProperty(@"ssh")]
-            public int SSPlus;
+            public int? SSPlus;
 
             [JsonProperty(@"ss")]
             public int SS;
 
             [JsonProperty(@"sh")]
-            public int SPlus;
+            public int? SPlus;
 
             [JsonProperty(@"s")]
             public int S;
 
             [JsonProperty(@"a")]
             public int A;
 
-            public int this[ScoreRank rank]
+            public int? this[ScoreRank rank]
```

Comment:
I'm unsure if doing this makes sense if it's going to have to get fixed up to 0 in places anyway (as it is being here in `UserBasedTable`).

Reply:
hmm, yeah, we can just make a null check in return